### ISSUE TITLE
sender.isCurrentPageHasErrors is sometimes wrong

### ISSUE THREAD

**Issue by cruckie** [surveyjs/surveyjs#871]
### Are you requesting a feature, reporting a bug or ask a question?
bug

### What is the current behavior?
sender.isCurrentPageHasErrors, on the onAfterRenderQuestion event, is randomly true/false in the provided example.

It worked in version 1.0.1, but no longer in 1.0.2. I might be a side effect of my previous bug report: https://github.com/surveyjs/surveyjs/issues/863

### What is the expected behavior?
sender.isCurrentPageHasErrors is true if and only if the form is not filled in correctly

### How would you reproduce the current behavior (if this is a bug)?
Use the provided example. I have some radiobuttons, each having a group of check boxes shown only when that radio button is chosen. The sender.isCurrentPageHasErrors is randomly true or false when I select different radio buttons, even though no check boxes (which are required) are selected.

1. Select radio button "Option 1". sender.isCurrentPageHasErrors is false, which is not expected.
2. I don't see the pattern from here, but select the different radio buttons a few times. The sender.isCurrentPageHasErrors is sometimes true and sometimes false even though no check boxes have been selected

#### Provide the test code and the tested page URL (if applicable)

Tested page URL:

Test code

```
Change "da-DK" to your language

 {
  "questionTitleTemplate": "{title} {require}",
  "showCompletedPage": false,
  "pages": [
    {
      "elements": [
        {
          "type": "radiogroup",
          "name": "1516180911717",
          "tag": "",
          "visibleIf": "",
          "isRequired": true,
          "title": {
            "da-DK": "Choose one"
          },
          "description": {
            "da-DK": ""
          },
          "choices": [
            {
              "id": "1516180920675",
              "value": "1",
              "text": {
                "da-DK": "Option 1"
              }
            },
            {
              "id": "1516180922668",
              "value": "2",
              "text": {
                "da-DK": "Option 2"
              }
            },
            {
              "id": "1516180935690",
              "value": "3",
              "text": {
                "da-DK": "Option 3"
              }
            }
          ]
        },
        {
          "type": "checkbox",
          "name": "1516181320439",
          "tag": "",
          "visibleIf": "{1516180911717} = \"1\"",
          "isRequired": true,
          "maxChoices": "",
          "title": {
            "da-DK": "Chose 1"
          },
          "description": {
            "da-DK": ""
          },
          "choices": [
            {
              "id": "1516181327767",
              "value": "1",
              "text": {
                "da-DK": "Option 1"
              }
            },
            {
              "id": "1516181329943",
              "value": "2",
              "text": {
                "da-DK": "Option 2"
              }
            }
          ]
        },
        {
          "type": "checkbox",
          "name": "1516181392422",
          "tag": "",
          "visibleIf": "{1516180911717} = \"2\"",
          "isRequired": true,
          "maxChoices": "",
          "title": {
            "da-DK": "Chose 2"
          },
          "description": {
            "da-DK": ""
          },
          "choices": [
            {
              "id": "1516181401542",
              "value": "1",
              "text": {
                "da-DK": "Option 1"
              }
            },
            {
              "id": "1516181406462",
              "value": "2",
              "text": {
                "da-DK": "Option 2"
              }
            }
          ]
        },
        {
          "type": "checkbox",
          "name": "1516181418582",
          "tag": "",
          "visibleIf": "{1516180911717} = \"3\"",
          "isRequired": true,
          "maxChoices": "",
          "title": {
            "da-DK": "Chose 3"
          },
          "description": {
            "da-DK": ""
          },
          "choices": [
            {
              "id": "1516181426678",
              "value": "1",
              "text": {
                "da-DK": "Option 1"
              }
            },
            {
              "id": "1516181427111",
              "value": "2",
              "text": {
                "da-DK": "Option 2"
              }
            }
          ]
        }
      ]
    }
  ]
}

```

### Specify your

* browser: Chrome 
* browser version: 63.0.3239.132
* surveyjs platform (angular or react or jquery or knockout or vue): jQuery
* surveyjs version: 1.0.2

**Comment by tsv2013** [comment]
Fixed via the 06728f8ad516db6e1c4826edaddba0ed0534b591 commit
Fix will be available in v1.0.3

**Comment by cruckie** [comment]
Thank you! It solves my issue.
